cirq/vis/heatmap.py

```diff
@@ -57,82 +60,81 @@ def relative_luminance(color: np.ndarray) -> float:
     return rgb.dot([0.2126, 0.7152, 0.0722])
 
 
-class Heatmap:
+def _get_qubit_row_col(qubit: QubitCoordinate) -> Tuple[float, float]:
+    if isinstance(qubit, grid_qubit.GridQubit):
+        return qubit.row, qubit.col
+    else:
+        return qubit[0], qubit[1]
+
+
+def _ave_qubit_row_col(qubitpair: QubitPair) -> Tuple[float, float]:
```

Comment:
1. nit avg is more typical for average
```suggestion
def _avg_qubit_row_col(qubitpair: QubitPair) -> Tuple[float, float]:
```
2.  I'd actually call this is more like "midpoint" - as that's the intention to calculate the mid point between them.